sdks/java/io/cdc/src/main/java/org/apache/beam/io/cdc/DebeziumOffsetHolder.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.io.cdc;
+
+import javax.annotation.Nullable;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class DebeziumOffsetHolder implements Serializable {
+    @Nullable public final Map<String, ?> offset;
+    @Nullable public final List<?> history;
+
+    DebeziumOffsetHolder(@Nullable Map<String, ?> offset, @Nullable List<?> history) {
+        this.offset = offset;
+        this.history = history == null ? new ArrayList<>() : history;
```

Comment:
Let's also add a configurable maximum fetched rows, so that we can write good end-to-end tests.